tracer/test/snapshots/StackExchangeRedisTests.V2_0_495.verified.txt

```diff
@@ -0,0 +1,4940 @@
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_323,
+    SpanId: Id_324,
+    Name: redis.command,
+    Resource: SUNIONSTORE,
+    Service: Samples.StackExchange.Redis-redis,
+    Type: redis,
+    Tags: {
+      component: StackExchangeRedis,
+      env: integration_tests,
+      out.host: stackexchangeredis,
+      out.port: 6379,
+      redis.raw_command: SUNIONSTORE StackExchange.Redis.Database.Set,
+      runtime-id: Guid_1,
+      span.kind: client
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_325,
+    SpanId: Id_326,
+    Name: redis.command,
+    Resource: TIME,
+    Service: Samples.StackExchange.Redis-redis,
+    Type: redis,
+    Tags: {
+      component: StackExchangeRedis,
+      env: integration_tests,
+      out.host: stackexchangeredis,
+      out.port: 6379,
+      redis.raw_command: TIME,
+      runtime-id: Guid_1,
+      span.kind: client
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_327,
+    SpanId: Id_328,
+    Name: redis.command,
+    Resource: TYPE,
+    Service: Samples.StackExchange.Redis-redis,
+    Type: redis,
+    Tags: {
+      component: StackExchangeRedis,
+      env: integration_tests,
+      out.host: stackexchangeredis,
+      out.port: 6379,
+      redis.raw_command: TYPE key,
+      runtime-id: Guid_1,
+      span.kind: client
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_329,
+    SpanId: Id_330,
+    Name: redis.command,
+    Resource: TYPE,
+    Service: Samples.StackExchange.Redis-redis,
+    Type: redis,
+    Tags: {
+      component: StackExchangeRedis,
+      env: integration_tests,
+      out.host: stackexchangeredis,
+      out.port: 6379,
+      redis.raw_command: TYPE StackExchange.Redis.Database.Key,
+      runtime-id: Guid_1,
+      span.kind: client
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_331,
+    SpanId: Id_332,
+    Name: redis.command,
+    Resource: UNLINK,
+    Service: Samples.StackExchange.Redis-redis,
+    Type: redis,
+    Tags: {
+      component: StackExchangeRedis,
+      env: integration_tests,
+      out.host: stackexchangeredis,
+      out.port: 6379,
+      redis.raw_command: UNLINK StackExchange.Redis.Database.Key,
+      runtime-id: Guid_1,
+      span.kind: client
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
```

Comment:
One surprise here was that this SPAN differed from v1_2_2. In v1_2_2, this span was `DEL StackExchange.Redis.Database.Key` but starting in this version it's `UNLINK StackExchange.Redis.Database.Key`. I don't think we need to note it anywhere but it was a weird find...

Reply:
Yeah, it looks to me that this was a bug introduced in the move from 1.x to 2.x, which was subsequently fixed in 2.0.571 ([Release Notes](https://stackexchange.github.io/StackExchange.Redis/ReleaseNotes#20571)):

> Fix: `KeyDeleteAsync` could, in some cases, always use `DEL` (instead of `UNLINK`)

The good news is not a bug on our side 😃